05_syntax_lab/01.py

```diff
@@ -1,5 +1,9 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+"""
+receive 10 numbers and print the largest of them
+"""
+max = int(raw_input())

```

Comment:
Fantastic! Just one comment-
Notice how github colours "max" in the same color as print (and differently from "number" and "Boom"). The reason is python has a function named max, so when you use that as a variable name it can be a bit confusing.